From d7748c1acb61fd65a8ddea99ecc714a10cbc7f4c Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Thu, 19 Oct 2023 09:53:47 +0500
Subject: [PATCH] Make builds read only

Apparently it doesnt work when this flag is decleared in the evo.mk file
---
 board-base.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/board-base.mk b/board-base.mk
index 2cb5dbc..b3500f3 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -10,3 +10,8 @@ BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs persist
 BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
+
+
+# Make builds RO
+BOARD_EXT4_SHARE_DUP_BLOCKS := true
+
-- 
2.42.0

